packages/build/src/log/messages/compatibility.js

```diff
@@ -146,6 +156,37 @@ export const logIncompatiblePlugins = function (logs, pluginsOptions) {
   logWarningArray(logs, incompatiblePlugins)
 }
 
+// TODO: This is temporary until we auto-install the Next runtime always for supported Next.js versions.
+const nextPluginVersionChecker = function (major, minor, patch) {
+  // maybe there's a semver package to check this
+
+  // version >= 4.0.0 && < 4.26.0
+  return (major === 4 && minor >= 0 && minor < 26) || (minor === 26 && patch === 0)
```

Comment:
I think there's a `@npm/semver` package used in this repo so better to use that?

```
// Think this semver range is correct for >= 4.0.0 but < 4.26.0
semver.satisfies('4.0.0 - 4.26.0')
```

Reply:
Make sure you set `includePrerelease` to true